Index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./Contents/index.css" />
+  <title>Mohammad Mehdi Pazouki</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <img src="./Contents/Resources/icons8-menu.svg" id="hamburger-menu" alt="menu" />
+    </nav>
+  </header>
+
+  <section id="intro">
+    <h1>Hi, I'm <b>Mohammad Mehdi Pazouki</b>, Happy to see you</h1>
+    <p>
+      Hello! I am a software developer! I can help you build a product,
+      feature or website. Taka a look of my works. If you like what you see
+      and have a project you need coded, don't hesitate and contact me.
+    </p>
```

Comment:
- [ ] Temple requires you have only your name as a header, and then 2 paragraphs with the presentation text.  Please make sure you correctly follow the template as close as possible.